### PR TITLE
updates to l/s/f json, language schema, language validate

### LINKED ISSUE #88: Create new field - DCNC Language Name
Move data of DCNC Notes in comment fields to "Language" field

### LINKED ISSUE #90: Studios/Facilities - Remove Null values
"Null" to be removed from Studios and Facilities. 

To be hard Coded on page builds.

### PR DESCRIPTION
**Pull request by SteveLLamb**
Closes #88, closes #90 

As per discussion in AdHoc
- New field added for DCNC Name in languages
- Updated schema and validate for Languages with new field added (also fixed minor canonical issues in both)
- Null values removed from Studio/Facilities.

### CODE REVIEW

File: src/main/data/languages.json [hunk 1]
```diff
@@ -67,624 +55,525 @@
...
+    "language": "Chinese - Taiwanese",
     "rfc5646Tag": "nan",
     "use": [
       "audio"
     ]
   },
   {
     "comments": [
-      "DCNC notes: (Audio only) - Chinese - Taiwanese Mandarin"
+      "(Audio only)"
```
Comment (palemieux) [inline src/main/data/languages.json:102]: This can be removed since `use` is `audio` only.

File: src/main/schemas/languages.schema.json [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
+      "comments": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
       },
       "dcncTag": {
         "type": "string"
       },
+      "language": {
```
Comment (palemieux) [inline src/main/schemas/languages.schema.json:19]: "dcncLanguage"

**Review by palemieux — CHANGES_REQUESTED** [review]

File: src/main/data/languages.json (hunk 1 above)
Comment (SteveLLamb) [reply src/main/data/languages.json]: Removed all audio only in notes where applicable, left subtitles only notes.

File: src/main/schemas/languages.schema.json (hunk 2 above)
Comment (SteveLLamb) [reply src/main/schemas/languages.schema.json]: updated field name in JSON, schema, and validation

**Review by palemieux — APPROVED** [review]

**Pull request merged**